docs/tutorial/work-pools.md

```diff
@@ -0,0 +1,335 @@
+    response.raise_for_status()
+    repo = response.json()
+    print(f"{repo_name} repository statistics 🤓:")
+    print(f"Stars 🌠 : {repo['stargazers_count']}")
+    print(f"Forks 🍴 : {repo['forks_count']}")
+
+
+if __name__ == "__main__":
+    get_repo_info.from_source(
+        source="https://github.com/discdiver/demos.git", 
+        entrypoint="repo_info.py:my_flow"
+    ).deploy(
+        name="my-first-deployment", 
+        work_pool_name="my-managed-pool", 
+    )
+```
+
+In the `from_source` method, we specify the source of our flow code.
+
+In the `deploy` method, we specify the name of our deployment and the name of the work pool that we created earlier.
+
+You can store your flow code in any of several types of remote storage.
+In this example, we use a GitHub repository, but you could use a Docker image, as you'll see in an upcoming section of the tutorial.
+Alternatively, you could store your flow code in cloud provider storage such as AWS S3, or within a different git-based cloud provider such as GitLab or Bitbucket.
+
+!!! note
+    In the example above, we store our code in a GitHub repository.
+    If you make changes to the flow code, you will need to push those changes to your own GitHub account and update the `source` argument of `from_source` to point to your repository.
+
+Run the script again and you should see a message in the CLI that your deployment was created with instructions for how to run it.
+
+<div class="terminal">
+
+```bash
+Successfully created/updated all deployments!
+
+                       Deployments                       
+┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━┓
+┃ Name                              ┃ Status  ┃ Details ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━┩
+│ get-repo-info/my-first-deployment  | applied │         │
+└───────────────────────────────────┴─────────┴─────────┘
+
+To schedule a run for this deployment, use the following command:
+
+        $ prefect deployment run 'get-repo-info/my-first-deployment'
+
+
+You can also run your flow via the Prefect UI: https://app.prefect.cloud/account/
+abc/workspace/123/deployments/deployment/xyz
+```
+
+</div>
+
+Navigate to your Prefect Cloud UI and view your new deployment.
+Click the "Run" button to trigger a run of your deployment.
+
+Because this deployment was configured with a Prefect Managed work pool, Prefect Cloud will run your flow on your behalf.
+
+View the logs in the UI.
+
+Now that you've updated your script, you can run it to register your deployment on Prefect Cloud:
+
+<div class="terminal">
+
+```bash
+python repo_info.py
+```
+
+</div>
+
+### Schedule a deployment run
+
+Now everything is set up for us to submit a flow-run to the work pool.
+Go ahead and run the deployment from the CLI or the UI.
+
+<div class="terminal">
+
+```bash
+prefect deployment run 'get_repo_info/my-deployment'
+```
+
+</div>
+
+Prefect Managed work pools are a great way to get started with dynamic infrastructure.  
```

Comment:
```suggestion
Prefect Managed work pools are a great way to get started with Prefect.  
```

Reply:
When a user uses a managed work pool they shouldn't have to think about infra at all.